### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by carl12**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,29 +4,20 @@
...
-
-- Node 6.13.0
-- etc
-
-## Development
+> npm run seed
+> npm run start
+> npm run start:server
 
 ### Installing Dependencies
```
Comment (MarissaShieh) [inline README.md]: It may make sense to move this section above the Usage section.

File: README.md [hunk 2]
```diff
@@ -37,3 +28,16 @@ npm install -g webpack
...
 
+## Server API
+
+Method | Path | Description| Request Body | Response Body
+----|----|----|----|----
+GET|api/chart/:stock-id|Gets all data for a stock| None | { stockInfo:{ relatedTags:[Str], stockCompany:Str, noOfOwners:Int, recommendationPercent:Int }, stockData:{ day:[Num], week:[Num], month:[Num], threeMonth:[Num], year:[Num], fiveYear:[Num] }, id:Num, averageStock:Num, changePercent:Num } |
+POST|api/chart/:stock-id/stock-price|Add new stock price for time given in body| {price:Num, time:DateTime} | None
+DELETE|api/chart/:stock-id|Delete all stock data associated with entry| None |None
+POST|api/chart|Create new stock with data defined in body| { stockInfo:{ relatedTags:[Str], stockCompany:Str, noOfOwners:Int, recommendationPercent:Int }, stockData:{ day:[Num], week:[Num], month:[Num], threeMonth:[Num], year:[Num], fiveYear:[Num] }, id:Num, averageStock:Num, changePercent:Num } | None
+PUT | api/chart/:stock-id/info/noOfOwners | Set new number of owners | Num | None
```
Comment (MarissaShieh) [inline README.md]: Why did you choose to have PUT endpoints for these but not for the other data like related tags, etc?

**Review by MarissaShieh — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (carl12) [reply README.md]: Good point, I have condensed my put requests into one request which overwrites the entire object. Updates to tags should be infrequent, so it makes sense to focus on other functionality.

File: README.md (hunk 1 above)
Comment (carl12) [reply README.md]: Good suggestion, readme is updated.

**Review by MarissaShieh — APPROVED** [review]

**Comment by MarissaShieh** [comment]
This looks good to go!

**Pull request merged**